ui/src/app/settings/components/GeneralForm/GeneralForm.test.js

```diff
@@ -0,0 +1,15 @@
+import { mount } from "enzyme";
+import React from "react";
+import GeneralForm from "./GeneralForm";
+
+describe("GeneralForm", () => {
+  let defaultProps = {
+    maasName: "bionic-maas",
+    enableAnalytics: true
+  };
+
+  it("can render", () => {
+    const wrapper = mount(<GeneralForm {...defaultProps} />);
```

Comment:
Please use `shallow` here instead, mounting the entire component tree makes the snapshots huge and of questionable usefulness.